focus_converter_base/focus_converter/conversion_configs/gcp/1_pricing_quantity_S001.yaml

```diff
@@ -1,4 +1,4 @@
-plan_name: Unnest usage.amount_in_pricing_units to get PricingQuantity
+plan_name: Unnest price.pricing_unit_quantity to get PricingQuantity
```

Comment:
Any reason why we are going with price.pricing_unit_quantity instead of usage.amount_in_pricing_units? price.pricing_unit_quantity is available only in detailed export and goes back only until Apr 2023.
https://cloud.google.com/billing/docs/how-to/export-data-bigquery-tables/detailed-usage

Reply:
Thank you, so we should use usage.amount_in_pricing_units.

This plan was suggested from a reference a long time ago so maybe not upto date.